frontend/src/components/elements/CodeBlock/CodeBlock.tsx

```diff
@@ -45,12 +45,12 @@ class CodeBlock extends React.PureComponent<Props> {
   public render(): React.ReactNode {
     if (this.props.language == null) {
       return (
-        <>
+        <div className="code-block">
```

Comment:
Can you change this to `stCodeBlock`? This is the pattern we use everywhere else